examples/agent/multi_document_agent.ts

```diff
@@ -23,7 +23,7 @@ Settings.llm = new OpenAI({ model: "gpt-4" });
 async function main() {
   await extractWikipedia(wikiTitles);
 
-  const countryDocs: Record<string, Document> = {};
+  const countryDocs: Record = {};
```

Comment:
nitpick: is there a problem with the type?

Reply:
I don’t remember why I changed this. Let me check what I’ve done here and I’ll let you know.

Reply:
Yup, I think this was my formatter being overzealous for some reason. I've changed in back.